### PR TITLE
Fill in email address from ocfEmail LDAP attribute

### LINKED ISSUE #22: email is not set when account is created
When a new staffer logs in for the first time, their account is automatically created but their email is not properly sent. That means that any actions such as creating tickets, replying, or commenting will cause emails to be rejected by the mail server. The email on your user should be automatically set when you log in.

Some discussion: https://irclogs.ocf.berkeley.edu/rebuild/2020-04-28#807371;

### PR DESCRIPTION
**Pull request by dkess**
Fixes #22

This is totally untested ¯\\_(ツ)_/¯

### CODE REVIEW

File: 99-ocf.pm [hunk 1]
```diff
@@ -14,6 +14,24 @@ Set($WebFallbackToRTLogin, 1);
...
+Set($ExternalSettings, {
+	'LDAP' => {
+		'type'             =>  'ldap',
+		'server'           =>  'ldap.ocf.berkeley.edu',
+		'base'             =>  'ou=People,dc=OCF,dc=Berkeley,dc=EDU',
+		'filter'           =>  '(objectClass=ocfAccount)',
+		'attr_match_list'  => [
+			'Name',
+		],
+		'attr_map' => {
```
Comment (kpengboy) [inline 99-ocf.pm:28]: Nit: what is the deal with this unique style of arrow formatting

Comment (dkess) [reply 99-ocf.pm:28]: Copied from the docs: https://docs.bestpractical.com/rt/4.4.4/RT/Authen/ExternalAuth.html

**Comment by cg505** [comment]
> [11] [Wed Apr 29 01:15:07 2020] [error]: Removing 'ldap' from ExternalInfoPriority, as it is not defined in ExternalSettings (/usr/share/request-tracker4/lib/RT/Config.pm:1196)

**Comment by cg505** [comment]
fixed that and got
> [17] [Wed Apr 29 01:33:28 2020] [critical]: RT::Authen::ExternalAuth::LDAP::_GetBoundLdapObj : Cannot connect to ldap.ocf.berkeley.edu (/usr/share/request-tracker4/lib/RT/Authen/ExternalAuth/LDAP.pm:649)

**Comment by dkess** [comment]
Thank you for testing! Added TLS configuration which is required for LDAP

**Comment by dkess** [comment]
yolo

**Pull request merged**